Woof/Woof/View/ReusableViews/TextEditorWithPlaceholder.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A view displaying TextEditor with a prompt text message for a user.
```

Comment:
Maybe the follow comment will be clearer. What do you think ?
```suggestion
/// A view that displays a text input field with a prompt text message for a user.
```

Reply:
Thanks for the suggestion. 
Fixed it.